src/Sulu/Bundle/SnippetBundle/Tests/Unit/Content/SnippetDataProviderTest.php

```diff
@@ -68,7 +75,8 @@ public function setUp()
             $this->snippetQueryBuilder->reveal(),
             $this->nodeHelper->reveal(),
             $this->proxyFactory->reveal(),
-            $this->documentManager->reveal()
+            $this->documentManager->reveal(),
+            $this->referenceStore->reveal()
```

Comment:
Maybe also check if the uuids have been correctly added to the reference store in the tests?